modules/web/src/app/node-data/basic/provider/nutanix/component.ts

```diff
@@ -165,7 +168,11 @@ export class NutanixBasicNodeDataComponent extends BaseFormValidator implements
       this.form.get(Controls.DiskSize).valueChanges
     )
       .pipe(takeUntil(this._unsubscribe))
-      .subscribe(_ => (this._nodeDataService.nodeData = this._getNodeData()));
+      .subscribe(_ => {
+        this._nodeDataService.nodeData = this._getNodeData();
+        const payload = this._getQuotaCalculationPayload();
+        this._quotaCalculationService.refreshQuotaCalculations(payload);
```

Comment:
We should also call this when `Categories` or `subnetName` change.

Reply:
Just FYI, I intentionally didn't added as existing valueChanges and inside payload it was directly getting subnets and templates but i have added to recalculate payload and trigger quotaCalculation instead of sep adding valueChanges for these 2 day controls